### PR TITLE
Reset _sysid and don't call pg_controldata when restore of backup in progress

### PR DESCRIPTION
**Pull request by CyberDem0n**
Otherwise there were some errors in a log from rest-api healthcheck endpoint

### CODE REVIEW

File: patroni/postgresql.py [hunk 1]
```diff
@@ -78,6 +78,7 @@ def __init__(self, config):
         self.replication_slots = []  # list of already existing replication slots
         self.retry = Retry(max_tries=-1, deadline=5, max_delay=1, retry_exceptions=PostgresConnectionException)
 
+        self._creating_replica = False
```
Comment (alexeyklyukin) [inline patroni/postgresql.py]: I think `create_replica_in_progress` would be a more meaningful name.

Comment (alexeyklyukin) [inline patroni/postgresql.py]: And it would be better to return this status via the API as well instead of sysid == 0 nonsense.

Comment (CyberDem0n) [inline patroni/postgresql.py]: Ok, I've found a better solution.
We should use `Postgresql.state` property for that. It is already exposed via REST API.
And regarding `sysid == 0`: if sysid is not defined  it is not exposed.

Comment (alexeyklyukin) [inline patroni/postgresql.py]: Good idea, so in addition to the running and stopped we should also have something like "initializing"

**Comment by alexeyklyukin** [comment]
👍

**Pull request merged**